### ISSUE TITLE
Attribute 'aria-label' on tag '<figure>'

### ISSUE THREAD

**Issue by 3Dprintt** [ckeditor/ckeditor5#18516]
Hello, I have a question about adding an image with a caption. When I save my content, Drupal generates a `<figure>` tag with a `role="figure"` attribute containing my image and a `<figcaption>`. The idea is that, in order to comply with the RGAA (French accessibility guidelines), an aria-label attribute should be added to the `<figure>` tag to respect this reference system. However, Drupal does not do this natively. What are the solutions besides using JS or a custom plugin? Does a solution already exist?

**Comment by Reinmar** [comment]
Drupal uses a modified version of CKEditor's plugin and saves a `<drupal-image>` element.

If you'd like to add the ARIA attribute for the markup rendered on the target page, AFAIR Drupal has a concept of filters. This may be a Drupal-native way to achieve this. But naturally, we're not here Drupal experts so it's better if you ask on Drupal channels.

If you'd like to add the ARIA attribute within the editor, then the only solution is a custom plugin for CKEditor (so JS). CKEditor supports concepts of model to view converters and you can extend existing ones.
